modal/_functions.py

```diff
@@ -1092,6 +1092,16 @@ def from_name(
         f = modal.Function.from_name("other-app", "function")
         ```
         """
+        if "." in name:
+            deprecation_warning(
+                (2025, 2, 5),
+                "Looking up class methods using Function.from_name will be deprecated"
+                " in a future version of Modal. Use modal.Cls.from_name() instead, e.g.\n"
+                "\n"
+                'DeployedClass = modal.Cls.from_name("some_app", "MyClass")\n'
```

Comment:
We should be able to translate the arguments into an exact suggestion, right?

Side note, we don't have a very consistent pattern in terms of what we suggest for naming the object that is returned by `Cls.from_name` and it's something I sort of get hung up on myself when writing code.